config.js

```diff
@@ -6,7 +6,9 @@
         flushInterval: 60000,
         stackdriver: {
             apiKey: process.env.STACKDRIVER_API_KEY || "api_key",
+			sourceFromPrefix: process.env.SOURCE_PREFIX_ENABLED=='true'? true: false,
```

Comment:
not exactly as environment variables are strings, and this value should be a boolean